scripts/scripts.js

```diff
@@ -0,0 +1,227 @@
+import * as dom from "./modules/dom-utils.js";
+import * as nf from "./modules/notesFunction.js";
+import * as test from "./modules/test.js";
+import * as gf from "./modules/generalFunctions.js";
+import * as cf from "./modules/calendarFunctions.js";
+
+dom.logAllEle(); // log all initially captured HTML elements
+
+/////////////////////////////////////////
+/* LIVE TIME */
+
+// Function to get current time & display
+function currentTime() {
```

Comment:
stick to using one type of functions so it's consistent across the project, maybe change to arrow function